MobileCenterCrashes/MobileCenterCrashes/MSCrashes.h

```diff
@@ -95,37 +95,23 @@ typedef NS_ENUM(NSUInteger, MSUserConfirmation) {
 ///-----------------------------------------------------------------------------
 
 /**
- *  Trap fatal signals via a Mach exception server.
+ * Disable to trap fatal signals via a Mach exception server.
```

Comment:
This is ambiguous.
How about "Disable the Mach exception handler. This can be useful when debugging." here and then add some more context. Further down.

```
The Mach exception handler executes in-process, and might interfere with debuggers when they attempt to suspend all active threads (which will include the Mach exception handler). Mach-based handling should _NOT_ be used when a debugger is attached. If you attach a debugger during runtime, this may cause issues if the Mach exception handler is enabled.

```